src/main/java/ru/skypro/homework/controller/CommentController.java

```diff
@@ -1,34 +1,58 @@
 package ru.skypro.homework.controller;
 
 import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
 import lombok.AccessLevel;
 import lombok.RequiredArgsConstructor;
 import lombok.experimental.FieldDefaults;
-import org.springframework.http.HttpStatus;
+import lombok.extern.log4j.Log4j2;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 import ru.skypro.homework.dto.AdsDTO;
 import ru.skypro.homework.dto.CommentDTO;
+import ru.skypro.homework.dto.CommentsDTO;
+import ru.skypro.homework.dto.UserDTO;
+import ru.skypro.homework.service.entities.CommentEntity;
+import ru.skypro.homework.service.entities.UserEntity;
+import ru.skypro.homework.service.impl.CommentServiceImpl;
+import ru.skypro.homework.service.repositories.CommentRepository;
 
-import javax.transaction.Transactional;
+import java.util.Optional;
 
+@Log4j2
 @CrossOrigin(value = "http://localhost:3000")
 @RequiredArgsConstructor
 @FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
 @RestController
 @RequestMapping("/ads")
 public class CommentController {
 
+    CommentRepository commentRepository;
+
+    CommentServiceImpl commentService;
```

Comment:
```suggestion
    CommentService commentService;
```
инъекцию надо делать по интерфейсам, а не реализациям